### PR TITLE
Replace `#plural` by `Formatter::pluralize`.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

File: Library/Homebrew/utils/formatter.rb [hunk 1]
```diff
@@ -90,4 +90,20 @@ def columns(*objects, gap_size: 2)
 
     output
   end
+
+  def pluralize(count, singular, plural = nil)
```
Comment (MikeMcQuaid) [inline Library/Homebrew/utils/formatter.rb]: I'd have probably flipped `count` and `singular` but don't have strong feelings.

File: Library/Homebrew/utils/formatter.rb [hunk 2]
```diff
@@ -90,4 +90,20 @@ def columns(*objects, gap_size: 2)
...
+    return "#{count} #{singular}" if count == 1
+
+    *adjectives, noun = singular.split(" ")
+
+    if plural.nil?
+      plural = {
```
Comment (MikeMcQuaid) [inline Library/Homebrew/utils/formatter.rb]: `plural ||=` instead?

File: Library/Homebrew/utils/formatter.rb (hunk 1 above)
Comment (reitermarkus) [inline Library/Homebrew/utils/formatter.rb]: I think it's more natural, since `pluralize(1, word)` -> `"1 word"`.

Comment (MikeMcQuaid) [inline Library/Homebrew/utils/formatter.rb]: 👍 to me

**Pull request merged**